ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -111,10 +111,25 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
     }
 
     /// Set `backgroundColor` to customize background color of the drawer
-    @objc open lazy var backgroundColor: UIColor = .init(dynamicColor: tokenSet[.drawerContentBackground].dynamicColor) {
-        didSet {
-            useCustomBackgroundColor = true
-            view.backgroundColor = backgroundColor
+    @objc open var backgroundColor: UIColor {
+        get {
+            let color: DynamicColor
+            if presentationController is UIPopoverPresentationController {
+                color = tokenSet[.popoverContentBackground].dynamicColor
+            } else if useNavigationBarBackgroundColor {
+                color = tokenSet[.navigationBarBackground].dynamicColor
+            } else {
+                color = tokenSet[.drawerContentBackground].dynamicColor
+            }
+            return UIColor(dynamicColor: color)
+        }
+        set {
+            guard let newColor = newValue.dynamicColor else {
+                return
+            }
+            tokenSet[.popoverContentBackground] = .dynamicColor { newColor }
+            tokenSet[.navigationBarBackground] = .dynamicColor { newColor }
+            tokenSet[.drawerContentBackground] = .dynamicColor { newColor }
```

Comment:
At this point, what's the purpose of getting these values independently? If they're all getting set to the same color, I mean

Reply:
They have different default values, and also if the client goes through tokens to override them they could just override a single one. But, if the client does `drawer.background = customColor`, we'll just set all the different backgrounds, to match the old behavior